src/routers/commentRouter.js

```diff
@@ -1,37 +1,12 @@
 import express from 'express';
-import threadRouter from './threadRouter.js';
+import * as commentController from '../controllers/commentController.js';
 
 const commentRouter = express.Router();
 
-// TODO:
-const createComment = async (req, res) => {
-  console.log('Server::createComment');
-};
-
-// TODO:
-const getComment = async (req, res) => {
-  console.log('Server::getComment');
-};
-
-// TODO:
-const updateComment = async (req, res) => {
-  console.log('Server::updateComment');
-};
-
-// TODO:
-const patchComment = async (req, res) => {
-  console.log('Server::patchComment');
-};
-
-// TODO:
-const deleteThread = async (req, res) => {
-  console.log('Server::deleteThread');
-};
-
-threadRouter.post('/', createComment);
-threadRouter.get('/:id', getComment);
-threadRouter.put('/:id', updateComment);
-threadRouter.patch('/:id', patchComment);
-threadRouter.delete('/:id', deleteThread);
+commentRouter.post('/', commentController.createComment);
```

Comment:
When u post a comment, u definitely need to post to a thread, right? Just creating a comment alone does not make sense. This whole API needs some upgrades.

Reply:
when I create a comment, a threadID is required for that comment to create. Doesn't that help to know where to create the comment?

Reply:
then it makes sense, since we will only allow creating a comment under a thread right? then the post request could fetch the thread id without sending it in the params.